intermediate-css-picasso-painting/index01.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```

Comment:
Students will build this project line by line, starting with the HTML. You could start the project here with no code and just a commented out section similar to the following:

```
<!-- 
    Begin by creating the structure of the page with the doctype and four main HTML elements:

    ```
    <!DOCTYPE html>
    <html>
    <head>
        <title></title>
    </head>
    <body>
    </body>
    </html>
    ```
-->
```

The next steps could be adding the title, then linking to the style sheets, and adding elements to the body of the page.